### PR TITLE
RC cli update

### PR DESCRIPTION
**Pull request by filipesilva**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [x] Other... Please describe: tests

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: integration/cli-hello-world-ivy-i18n/e2e/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+### legacy
+
+The legacy `ng xi18n` tool extracts the messages from the Angular templates, into the XLIFF 1.2
+format with legacy message ids (`tmp/legacy-locales/messages.legacy.xlf`).
+
+The translation file is modified to apply a simple translation.
+
+The app must be compiled using the `i18nLegacyMessageIdFormat` option set to ensure that the correct
+message ids are used to match those in the translation files.
```
Comment (petebacondarwin) [inline integration/cli-hello-world-ivy-i18n/e2e/README.md]: This is actually the default for v9 - as long as the `i18nFormat` is passed to the compiler.

File: integration/cli-hello-world-ivy-i18n/e2e/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+The app is translated using the compile-time inlining tool to generate a copy that has the
+translated message in it.
+
+## Hosting
+
+Since the CLI hosts from and in-memory file-system the compile-time inliner is not able to
```
Comment (petebacondarwin) [inline integration/cli-hello-world-ivy-i18n/e2e/README.md]: typo:

> From an~~d~~ in-memory

**Review by petebacondarwin — APPROVED** [review]

File: integration/cli-hello-world-ivy-i18n/e2e/README.md (hunk 1 above)
Comment (clydin) [reply integration/cli-hello-world-ivy-i18n/e2e/README.md]: The CLI also automatically sets this for the compiler based on the input translation file.

**Review by clydin — APPROVED** [review]

File: integration/cli-hello-world-ivy-i18n/e2e/README.md (hunk 2 above)
Comment (filipesilva) [reply integration/cli-hello-world-ivy-i18n/e2e/README.md]: Same as above.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-srcs",
-  "version": "9.0.0-next.15",
+  "version": "9.0.0-next.19",
```
Comment (josephperrott) [inline package.json]: We never released a `next.19` for framework, this should be `.15` correct?

**Review by josephperrott — APPROVED** [review]
LGTM, one small comment

**Comment by AndrewKushnir** [comment]
FYI, adding "cleanup" label to this PR after discussing it with Vikram (since this PR is in progress of switching to RC version). Thank you.

**Comment by filipesilva** [comment]
@AndrewKushnir updated to use CLI RC. I think it's now good to go.

**Review by devversion — APPROVED** [review]

**Pull request closed without merging**